### ISSUE TITLE
update min/max for range slider?

### ISSUE THREAD

**Issue by indus** [tweakpane/plugin-essentials#12]
I want to update the min/max of the range slider at runtime.
I've tried `sc.props.set("minValue", min);` But it looks like this function is only available for regular sliders and not for range sliders.
Would it be possible to add a feature like https://github.com/cocopon/tweakpane/issues/360 for the range slider as well?

**Comment by cocopon** [comment]
Released the latest version 0.1.7, and now you can change its range like this:

```ts
const i1 = pane.addInput(PARAMS, 'foobar', {
  view: 'interval',
  min: 0,
  max: 100,
});

// Note that it uses a private field and may break in future updates
const sliderProps = (i1.controller_.valueController as any).sc_.sliderProps;
sliderProps.set('minValue', 1);
sliderProps.set('maxValue', 10);
```

**Comment by indus** [comment]
@cocopon will come in use immediately 😀
